Gopkg.toml

```diff
@@ -85,7 +85,7 @@ required = [
   name = "github.com/giantswarm/ipam"
 
 [[constraint]]
-  version = "~0.1.0"
+  branch = "legacy-fix-conntrack"
```

Comment:
I'll bump this to 0.1.1 once https://github.com/giantswarm/k8scloudconfig/pull/712 is merged/released.